ui/webui/webui-desktop

```diff
@@ -66,21 +66,23 @@ esac
 
 # prepare empty firefox profile dir with theme based on the passed profile id
 FIREFOX_THEME_DIR="/usr/share/anaconda/firefox-theme"
-FIREFOX_PROFILE_PATH="/tmp/anaconda-firefox-profile"
+LIVEUSER=$(id -n -u $PKEXEC_UID)
```

Comment:
So this same script is also used on the boot.iso to run the Web UI there & I don't think there is a live user account available there. I'll try to build a boot.iso with this PR & also a Live image when I'm at it.

Reply:
should be okay, if $PKEXEC_UID is unset LIVEUSER should just get set to the current running user.

Maybe the variable LIVEUSER could have a better name (INSTALLER_USER?)

Reply:
of course that means firefox continues to run as root on boot.iso, which is less than ideal, but it shouldn't complain in that case, because it won't think it's a user elevated to root, but root all the way through

Reply:
Yes, I think renaming the variable would be best, plus maybe a comment we expect it to be liveuser on live?

Reply:
> Yes, I think renaming the variable would be best, plus maybe a comment we expect it to be liveuser on live?

Agreed with both.